### PR TITLE
Enabling auto-update for CSS-Mint

### PR DESCRIPTION
**Pull request by ArunMichaelDsouza**

### CODE REVIEW

File: files/css-mint/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "CSS-Mint",
+  "repo": "ArunMichaelDsouza/CSS-Mint",
+  "files": {
+    "include": ["./build/css-mint.min.css"]
```
Comment (jimaek) [inline files/css-mint/update.json]: `build/css-mint.min.css` would be the correct path

**Comment by ArunMichaelDsouza** [comment]
Updated @jimaek

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by ArunMichaelDsouza** [comment]
@jimaek how long does it take for a project to get updated ?

**Comment by ArunMichaelDsouza** [comment]
Well that was instant! Thanks anyways. :+1:

**Comment by jimaek** [comment]
:)

**Pull request closed without merging**